testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergSparkCompatibility.java

```diff
@@ -186,12 +207,15 @@ public void testSparkReadsTrinoPartitionedTable()
         onTrino().executeQuery("DROP TABLE " + trinoTableName);
     }
 
-    @Test(groups = {ICEBERG, PROFILE_SPECIFIC_TESTS})
-    public void testTrinoReadsSparkPartitionedTable()
+    @Test(groups = {ICEBERG, PROFILE_SPECIFIC_TESTS}, dataProviderClass = TestingIcebergStorageFormat.class, dataProvider = "storageFormats")
+    public void testTrinoReadsSparkPartitionedTable(TestingIcebergStorageFormat storageFormat)
     {
-        String baseTableName = "test_spark_reads_trino_partitioned_table";
+        String baseTableName = "test_spark_reads_trino_partitioned_table_" + storageFormat.name();
```

Comment:
pre-existing: table name != test intention

Reply:
Yes, the existing table name is reversed from the test's name